__tests__/app.test.js

```diff
@@ -315,3 +315,41 @@ describe('/api/articles/:article_id/comments', () => {
         });
     });
 });
+
+describe('/api/comments/:comments_id', () => {
+    describe('DELETE', () => {
+        test('DELETE:204 should delete a comment using given parameterised comment_id', () => {
+            const removedCommentId = 1;
```

Comment:
There really is no need to have this separate up here, this is not a dynamic test.